app/collections/show.cjsx

```diff
@@ -8,11 +8,15 @@ Avatar = require '../partials/avatar'
 Loading = require '../components/loading-indicator'
 HandlePropChanges = require '../lib/handle-prop-changes'
 TitleMixin = require '../lib/title-mixin'
+classNames = require 'classNames'
 
 counterpart.registerTranslations 'en',
   collectionPage:
     settings: 'Settings'
     collaborators: 'Collaborators'
+    collectionsLink: '%(user)s\'s\u00a0Collections'
+    favoritesLink: '%(user)s\'s\u00a0Favorites'
+    userLink: '%(user)s\'s\u00a0Profile'
```

Comment:
Oh, sure, i had no idea your comment was anything to do with `\u00a0`. 
`whitespace: nowrap` is a good idea, will use that.